tests/test_hinfo.py

```diff
@@ -0,0 +1,74 @@
+import os
+import sqlite3
+import tempfile
+from pathlib import Path
+
+import pytest
+import yaml
+from lsst.consdb import hinfo, utils
+from lsst.resources import ResourcePath
+from sqlalchemy import MetaData, Table, select
+
+
+@pytest.fixture
+def tmpdir(scope="module"):
+    os.environ["INSTRUMENT"] = "LATISS"
+    tmpdir = Path(tempfile.mkdtemp())
+    return tmpdir
+
+
+@pytest.fixture
+def engine(tmpdir, scope="module"):
```

Comment:
Just for my own context & learning, this looks like it could be relevant in other test files where we need to create an sqlite engine. Do other test files exist yet that need that, should we generalize this one, should we just keep an eye on it for further test development?